pkg/client/url/url.go

```diff
@@ -0,0 +1,29 @@
+// Copyright 2025 The Perses Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package url
+
+import (
+	"net/url"
+	"path"
+)
+
+func CloneURLAddingPath(u *url.URL, sections ...string) *url.URL {
```

Comment:
I would put that in `pkg/api/v1/common/url.go` instead and rename it by

```suggestion
func NewURL(url *URL, paths ...string) *URL {
```

Reply:
I did the changes as Celian doesn't have more time to spend on this fix.